tensorflow/contrib/tensor_forest/kernels/stats_ops.cc

```diff
@@ -275,10 +291,22 @@ class ProcessInputOp : public OpKernel {
     // with mutexes.
     std::unordered_map<int, std::unique_ptr<mutex>> locks;
     std::unordered_map<int32, std::vector<int>> leaf_examples;
+    std::unordered_map<int32, std::vector<int>> leaf_random_features;
+    int32 num_total_features = input_spec_.dense_features_size();
+    // Since num_splits_to_consider is a constant here, we set the depth as 0
+    int num_splits_to_consider =
+        ResolveParam(param_proto_.num_splits_to_consider(), 0);
```

Comment:
Pardon, why such a complicated way?  wouldn't param_proto_.num_splits_to_consider() work?

Reply:
this is somehow previous implementation, i just copied it here, but you are right, i'll change here